### ISSUE TITLE
Updating version of OpenZFS-on-Windows

### ISSUE THREAD

**Issue by sskras** [openzfsonwindows/openzfs#266]
This probably belongs more to a QnA / Discussions section (which I miss in this repo).

I have installed `2.1.6rc3`:
```
saukrs@DESKTOP-O7JE7JE MSYS ~
$ sudo zfs version
zfswin-2.1.6rc3
zfs-kmod-zfswin-2.1.6rc3
```

What is the reliable way to update this thing to Eg. newest `2.2.0rc4` ?

Should I just download [`OpenZFSOnWindows-debug-2.2.0rc4.exe`](https://github.com/openzfsonwindows/openzfs/releases/download/zfswin-2.2.0rc4/OpenZFSOnWindows-debug-2.2.0rc4.exe) and run the installer?

**Comment by lundman** [comment]
I believe that should work, if it doesn't, I'll have to fix something :)

**Comment by sskras** [comment]
OK, I went without uninstalling the old version before that.

It asked me if would like to reboot just now.  Is necessary to reboot the OS to activate the new driver?

**Comment by lundman** [comment]
Yeah maybe. Not sure, you can unload it if you export everything first, but to load a new one, Windows seems to want to reboot. Might be a trick to it, I am unsure.

**Comment by sskras** [comment]
Thanks for the insights.

I going to keep the issue open for a while.  It would serve as a reminder to do some RnD about updates without reboot (I have no pools imported mounted at the moment).

But if you wish, feel free to close it.

**Comment by sskras** [comment]
Maybe this should be converted into a discussion? (To keep the list cleaner:)
https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion
